config.json

```
@@ -37,6 +37,44 @@
         }
       }
     },
+    "cnpm": {
+      "default": "9.3.2+sha1.05a307b7c1d34a368b8d30ff159dfd0533c8c328",
+      "fetchLatestFrom": {
+        "type": "url",
+        "url": "https://registry.npmmirror.com/cnpm",
+        "fields": {
+          "tags": "latest",
+          "versions": "tags"
+        }
+      },
+      "transparent": {
+        "commands": [
+          [
+            "cnpm",
+            "sync"
+          ]
+        ]
+      },
+      "ranges": {
+        "*": {
+          "url": "https://cdn.npmmirror.com/packages/cnpm/{}/cnpm-{}.tgz",
+          "bin": {
+            "cnpm": "./bin/cnpm"
+          },
+          "commands": {
+            "use": ["cnpm", "-v"]
```

```suggestion
            "use": ["cnpm", "install"]
```
The other package managers use `install` here, could `cnpm` do that as well so the behaviour of `corepack use cnpm` matches the others?